Integrations/integration-Fortigate.yml

```diff
@@ -0,0 +1,1381 @@
+        service_name =  demisto.args().get('serviceName')
+        tcp_range =  demisto.args().get('tcpRange', '')
+        udp_range =  demisto.args().get('udpRange', '')
+
+        response = create_firewall_service_request(service_name, tcp_range, udp_range)
+
+        contents.append({
+            'Name': service_name,
+            'Ports': {
+                'TCP': tcp_range,
+                'UDP': udp_range
+            }
+        })
+        service_context.append({
+            'Name': service_name,
+            'Ports': {
+                'TCP': tcp_range,
+                'UDP': udp_range
+            }
+        })
+
+        context['Fortigate.Service(val.Name && val.Name === obj.Name)'] = service_context
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': contents,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('FortiGate firewall service ' + service_name + ' created successfully', contents),
+            'EntryContext': context
+        })
+
+
+    def create_firewall_service_request(service_name, tcp_range, udp_range):
+        uri_suffix = 'cmdb/firewall.service/custom/'
+        if does_path_exist(uri_suffix + service_name):
+            return_error('Firewall service already exists.')
+        payload = {
+            'name': service_name,
+            'tcp-portrange': tcp_range,
+            'udp-portrange': udp_range
+        }
+        response = post_request(uri_suffix, {}, json.dumps(payload))
+        return response
+
+
+    def get_policy_command():
+        contents = []
+        context = {}
+        policy_context = []
+        policy_name = demisto.args().get('policyName')
+        policy_id = demisto.args().get('policyID')
+        policy_title = 'all policies'
+
+        policies = get_policy_request(policy_id)
+
+        for policy in policies:
+            if policy_name == policy.get('name') or not policy_name:
+                if policy_name or policy_id:
+                    policy_title = policy.get('name')
+                security_profiles = []
+                all_security_profiles = [ policy.get('webfilter-profile'), policy.get('ssl-ssh-profile'), policy.get('dnsfilter-profile'),                                                                                                     policy.get('profile-protocol-options'), policy.get('profile-type'), policy.get('av-profile') ]
+                for security_profile in all_security_profiles:
+                    if security_profile:
+                        security_profiles.append(security_profile)
+
+                contents.append({
+                    'Name': policy.get('name'),
+                    'ID': policy.get('policyid'),
+                    'Description': policy.get('comments'),
+                    'Status': policy.get('status'),
+                    'Source': policy.get('srcaddr')[0].get('name'),
```

Comment:
Are we sure this won't fail? Cause if `None` is returned from first `get`, it might fail when trying to access `[0]`